src/main/java/it/smartcommunitylab/aac/oidc/provider/OIDCAuthenticationProvider.java

```diff
@@ -253,6 +253,15 @@ public Authentication doAuthenticate(Authentication authentication) throws Authe
 
             return auth;
         } catch (OAuth2AuthenticationException e) {
+            logger.debug(
+                String.format(
```

Comment:
no need to format as string for the logger, just prepare the message "error ... {}" and then pass parameters. Drop the throwable.